File: release/fixtures/full-compose/app/src/main/java/com/tongsr/kaleido/matrix/compose/MainActivity.java
package com.tongsr.kaleido.matrix.compose;

import android.app.Activity;
import android.os.Bundle;
import android.util.Log;

public final class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle state) {
        super.onCreate(state);
        Log.i("KaleidoRuntimeProbe", "KALEIDO_PROBE_PASS resource=AppTheme compose=consumer");
    }
}

File: release/fixtures/kotlin-safe/app/src/main/kotlin/com/tongsr/kaleido/matrix/kotlin/MainActivity.kt
package com.tongsr.kaleido.matrix.kotlin

import android.app.Activity
import android.os.Bundle
import android.util.Log

class MainActivity : Activity() {
    override fun onCreate(state: Bundle?) {
        super.onCreate(state)
        Log.i("KaleidoRuntimeProbe", "KALEIDO_PROBE_PASS resource=AppTheme")
    }
}
